modules/sdk/project-templates/project-templates-spring-mvc-portlet/src/main/resources/archetype-resources/build.gradle

```diff
@@ -23,13 +23,20 @@ buildscript {
 apply plugin: "com.liferay.css.builder"
 
 dependencies {
-#if (${framework.equals("portletmvc4spring")} && ${frameworkDependencies.equals("embedded")})
+#if (${jakartaCompatible.equals("true")} && ${frameworkDependencies.equals("embedded")})
+	implementation group: "com.liferay.portletmvc4spring", name: "com.liferay.portletmvc4spring.framework", version: "6.0.0-M1"
```

Comment:
Let's follow the same pattern as the `*Version` variables above, since in many places it seems this version is re-used.